### ISSUE TITLE
Expand the README to be a useful base for external extension pages

### ISSUE THREAD

**Issue by Zegnat** [indieweb/verify-me#7]
I just received an additional (after approval) reviewer reply on [the AMO version of Verify Me](https://addons.mozilla.org/en-US/firefox/addon/verify-me/) requesting a more clear description of what the extension does. I added the following:

> Looks for [rel="me" links](https://indieweb.org/rel-me) on the current page, then check if they link back with a rel="me" too. Shows a green ✔️ or red ❌ depending on whether they do or not. Allowing [distributed verification](http://www.kevinmarks.com/distributed-verify.html) of ownership and profile equivalence between different websites and social media profiles.

It would be great if we can work on the README in this repository to contain all the information necessary for external portals like these. That way a simple copy-paste is all that will be needed for future releases.

* [ ] A single line summary of the extension.
   
   AMO summaries are max 250 characters, but shorter seems better. Does the Chrome Web Store have something like it, what is its limit? GitHub has the repo description up top. Not sure if that comes with any length limitations, but GitHub-likes such as Gogs have imposed limits around 250 characters too.

* [ ] A multi paragraph description of the extension.
   
   What does it do, and why would you use it? More so than with the single line summary, a user should be able to decide whether this is worth installing based on the description.
   
   AMO recommends 250 characters, but doesn’t impose any special limit (that is obvious to me). It also supports putting links in here. I think this would be the “Overview” section on the Chrome Web Store? Does that one support external links?
   
* [ ] Screenshots of the extension in use.
   
   Both AMO and the Chrome Web Store want screenshots. I think these should be put in the repository and also included in our own README. AMO has the following recommendations for screenshots:
   
   > The maximum and recommended size for screenshots is 1280x800 pixels. 640x400 pixels is a good alternative if you only have smaller images.

**Comment by grahamperrin** [comment]
When I first saw the AMO page (before this issue was raised) IIRC the first thing I wondered was: 

* is this for an end user, or for a site developer?

> AMO summaries are max 250 characters, but shorter seems better. 

Yes and no. 

Imagine your summary listing appearing in the midst of 15 or so others. 

https://addons.mozilla.org/firefox/search/?type=extension&page_size=100&q=link%20checker

https://addons.mozilla.org/firefox/search/?type=extension&page_size=100&q=check%20links

Make it easy not only for people who seek your extension (or something like it); make it easy for people who seek something *else* to know – at a glance – that yours isn't a match. 

In the long description, ELI5 if you can. Something in plain English that captures the essence of https://indieweb.org/rel-me or whatever. 

----

For myself, I found a few links …

* [rel="me"](https://www.slideshare.net/emilylewis/microformats-web-semantics-more/14-relme_Microformats_Web_Semantics_More) at slide 14 of _Microformats: Web Semantics & More_ (2009)
* [Official Google Webmaster Central Blog [EN]: Authorship markup and web search](https://webmasters.googleblog.com/2011/06/authorship-markup-and-web-search.html) (2011)
* [?rel=me ?rel=author ?rel=really, Google? | How to Make, Market and Sell Ebooks](https://ebooksuccess4free.wordpress.com/2012/01/17/relme-relauthor-relreally-google/) (2012)
* [A briefing on Using 'Rel=me' for SEO | Smart Insights](https://www.smartinsights.com/search-engine-optimisation-seo/index-inclusion/google-rel-me/) (2013)
* [Google No Longer Recommends rel=me Microformats](https://www.seroundtable.com/google-rel-me-schema-25636.html) (2018)
* [Mastodon Now Supports Rel=Me – Interdependent Thoughts](https://www.zylstra.org/blog/2018/10/mastodon-rel-me/) (2018)

**Comment by Zegnat** [comment]
I knew about some of those articles, and wrote [a reply to one of them](https://wiki.zegnat.net/media/the-real-deal-about-rel-me.html) that did fairly well on Hacker News.

The problem with a lot of them are that they are about SEO and what Google is and isn’t doing, while I have never seen a single resource saying Google ever actually used `rel="me"` for any of its indexing at any point in time. So I would try and avoid those sources completely.

**Comment by grahamperrin** [comment]
Thanks :+1: 

> make it easy for people who seek something else to know – at a glance – that yours isn't a match.

I stumbled across a good example, https://addons.mozilla.orgfirefox/search/?q=editor%20review I can tell at a glance that the most relevant result is not for me (I'm not an AMO reviewer)
